scikits/fitting/__init__.py

```diff
@@ -77,14 +77,17 @@
 from spline import *
 
 
+logger = logging.getLogger(__name__)
+
+
 def _setup_test():
     """Create test() method that will run unit tests via nose."""
     args = ['', '--exe', '-w', __path__[0]]
     try:
         import nose as _nose
     except ImportError:
         def test():
-            print('Could not load nose. Unit tests not available.')
+            logger.warning('Could not load nose. Unit tests not available.')
```

Comment:
I'm not sure exactly how this is intended to be used, but if the logging module hasn't been set up all the user will see is a message that there are no handlers (or possibly nothing at all, I forget). Maybe the `warnings` module is a better fit here?